.github/workflows/ci.yml

```diff
@@ -0,0 +1,128 @@
+# This is a the GitHub Action to run the CI for SODA Dashboard
+
+name: SODA Installer CI
+
+# Controls when the action will run. 
+# Triggers the workflow on push or pull request events but only for the main branch
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        go-version: [1.13.x]
+        os: [ubuntu-16.04]
```

Comment:
please include 18  too

Reply:
The current CI was running only on Ubuntu 16.04 Xenial.
The ansible setup with GitHub Actions has an issue when using Ubuntu18.04.
I will debug that and make the change when it is working. Meanwhile we can continue using the CI with 16.04 as before.

Reply:
ok
Just open a tracker for this

Reply:
I have found the solution and fixed it. the CI now works for Ubuntu18.04 as well. 
I will be pushing more commits now for the fix.